requirements.txt

```diff
@@ -9,7 +9,7 @@ requests
 setuptools
 types-dataclasses
 typing-extensions>=4.8.0
-sympy
+sympy >= 1.13.0
```

Comment:
You will need to pin the version in `setup.py` and CI constraint files. See https://github.com/pytorch/pytorch/pull/130836/files for reference.

Reply:
Yes I was not planning on merging this one as-is and this is more to use as an end-to-end check that CI works.
I will rebase above both the pybind and sympy upgrades once they're both landed!

Thanks a lot for your help with all of this!